samples/notebooks/Multiple outputs - basics.ipynb

```diff
@@ -0,0 +1,206 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multiple outputs example\n",
+    "This notebook is a simple example of how to make a component with multiple outputs using the Pipelines SDK.\n",
+    "\n",
+    "## Before running notebook:\n",
+    "\n",
+    "### Setup notebook server\n",
+    "This pipeline requires you to [setup a notebook server](https://www.kubeflow.org/docs/notebooks/setup/) in the Kubeflow UI.  After you are setup, *upload the notebook in the Kubeflow UI* and then run it in the notebook server.\n",
+    "\n",
+    "### Create a GCS bucket\n",
+    "This pipeline requires a GCS bucket.  If you haven't already, [create a GCS bucket](https://cloud.google.com/storage/docs/creating-buckets) to run the notebook.  Make sure to create the storage bucket in the same project that you are running Kubeflow on to have the proper permissions by default.  You can also create a GCS bucket by running `gsutil mb -p <project_name> gs://<bucket_name>`.\n",
+    "\n",
+    "### Upload the notebook in the Kubeflow UI\n",
+    "In order to run this pipeline, make sure to upload the notebook to your notebook server in the Kubeflow UI.  You can clone this repo in the Jupyter notebook server by connecting to the notebook server and then selecting New > Terminal.  In the terminal type `git clone https://github.com/kubeflow/pipelines.git`.\n",
+    "\n",
+    "### Install Kubeflow pipelines\n",
+    "Install the `kfp` package if you haven't already."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!pip install kfp --upgrade"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup project info and imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "GCS_BUCKET = 'gs://[BUCKET-NAME]' # GCS bucket name\n",
```

Comment:
Could you add a tag as "parameters" for this cell such that later on we can parameterize the notebook and have it tested in the sample testing infra. Example is https://github.com/kubeflow/pipelines/blob/master/samples/notebooks/KubeFlow%20Pipeline%20Using%20TFX%20OSS%20Components.ipynb
You can edit the notebook tag in your notebook environment.

Reply:
Updated